.travis.yml

```diff
@@ -0,0 +1,34 @@
+dist: trusty
+sudo: required
```

Comment:
I've been defaulting to the [trusty beta](https://docs.travis-ci.com/user/trusty-ci-environment/) image after struggling to find a travis configuration with a reasonable cc toolchain for the rules_protobuf repo I've been maintaining (specifically, getting grpc++ and boringssl to compile required this).  

For rules_go, it may not be required. I'll test your jsonnet model and use that if it works.